speaking-portal-project/src/phonemeFactory/mouthCueProcessor.ts

```diff
@@ -0,0 +1,39 @@
+import { MouthCue, MouthCueArray } from '../types'
+import fs from 'fs'
+
+//TODO: Find ways to optimize this
+export async function mouthCuesToInputFile({
+  mouthCues,
+  outputPath = '../ffmpeg/input.txt',
+}: {
+  mouthCues: MouthCue[]
+  outputPath?: string
+}) {
+  try {
+    await fs.promises.writeFile(outputPath, generateMouthData(mouthCues), {
+      flag: 'w',
+    })
+    /* Lines 17-20 for testing purposes only
```

Comment:
We could create a 'testing' mode for project that could trigger logs only during testing purposes. Not something for now, but something we could add later

Reply:
Yeah I was thinking something like that could be good! that's why I left the commented out stuff in for now